### DISCUSSION TITLE
Hello. I am a newbie.

### DISCUSSION THREAD

**Discussion by toomsie** [jquery/jquery#5221] [General]
Hello everyone!

My name is Gerald, and I'm excited to be joining the jQuery community. I've been using jQuery for while learning JavaScript and have found it to be an incredibly useful tool for web development.

I'm interested in contributing to the project and would love to help out in any way I can.  I have experience with most modern programming languages and some QA testing experience, and I'm eager to learn more and contribute to the community.

Thank you for welcoming me to the jQuery community, and I look forward to working with all of you!
Best regards,
Gerald

**Comment by timmywil** [comment]
Thanks for your interest in contributing! The [Contributing Guide](https://github.com/jquery/jquery/blob/main/CONTRIBUTING.md) is a good place to start if you're interested. There's also https://contribute.jquery.org/.

Some of the info is a little outdated, but there's a lot of great information on how to get started.
